tools/cooja/apps/collect-view/build.gradle

```diff
@@ -0,0 +1,18 @@
+apply plugin: "java"
+
+libsDirName = ".."
+
+repositories {
+    jcenter()
+}
```

Comment:
Indeed it's needed because you could have different repositories. The dependencies don't bring a specific repository with them.